shared/fs/filepreview/text-view.desktop.js

```diff
@@ -21,18 +21,19 @@ const TextView = (props: TextViewProps) => (
 // provided by <FilePreview/>
 const webviewCSS = `
 html{
-  background-color: ${globalColors.blue5};
-  padding-top: ${globalMargins.medium}; 
-  padding-bottom: ${globalMargins.medium}; 
+  background-color: ${Styles.globalColors.blue5};
+  padding-top: ${Styles.globalMargins.medium}; 
+  padding-bottom: ${Styles.globalMargins.medium}; 
   margin: 0;
 }
 body{
-  background-color: ${globalColors.white};
-  padding: ${globalMargins.xlarge};
+  background-color: ${Styles.globalColors.white};
+  padding: ${Styles.globalMargins.xlarge};
   margin: 0;
-  color: ${globalColors.black_50};
+  color: ${Styles.globalColors.black};
   line-height: 1.38;
   font-size: 13;
```

Comment:
I think we could do 14 here, I tried your branch and it's still very hard to read in 13.